### ISSUE TITLE
Timeouts on Pantheon Doc Update 

### ISSUE THREAD

**Issue by rakibul-islam-milon** [pantheon-systems/documentation#5743]
Re: [Timeouts on Pantheon](https://pantheon.io/docs/timeouts)

Priority: Low 

## Issue Description:
Further clarify or add note that scripts would only run up to 59 seconds and reasons for such a short timeout in the platform

## Suggested Resolution

- It may be confusing as there are references to 120 secs on other timeouts but maybe a quick note that the safest that a script that can run is 59 sec

- Maybe add a good reason why the platform has a short timeout eg. It is needed for it to easily horizontally scale quickly and not wait for long processes to finish

**Comment by carl-alberto** [comment]
Nice issue raised @rakibul-islam-milon. This may be a good addition to raise in this doc so we can easily refer customers later when customers ask such a short timeout compared to other hosts.

**Comment by alexfornuto** [comment]
@rakibul-islam-milon thanks for the issue! Are you talking about local scripts that invoke connections, or scripts run on the platform? If the latter, what kind of scripts? This additional context will help us proceed.

**Comment by twfahey1** [comment]
Would like to hear more from @rakibul-islam-milon , but I found this issue attempting to clarify the current docs page, specifically "Connection Timeout" and "PHP max_execution_time". This may be too specific for docs as is, but as an example, let's say we have the following function:
```
/**
 * Test a long running script
 */
function test_long_running_script() {
  echo "Processing long running operation, will take ~115 seconds.";
  sleep(115);
  echo "All done!";
}
```
If this function were to be invoked in, for example, a Controller callback to render a page at `/long-running-test`, it would fail with a timeout, due to the 59 second limit on scripts running through the GCDN. However, if we were to invoke this function via a drush command, or a quicksilver hook, it would succeed. 

I realize that this can be inferred from the current docs, but perhaps it might somehow be worked in somewhere as a more concrete example of the limitations and possibilities for a developer coming to the platform.

**Comment by alexfornuto** [comment]
@twfahey1 sounds like you were already on your way to making a PR to clarify this area of the doc. Maybe you can tackle this issue as well? I _mostly_ understand what you're saying, but as I'm not a CMS dev, the situational description of "a Controller callback" doesn't mean much to me.

**Comment by joa-pan** [comment]
Addressed in PR #6842
